articles/virtual-machines/linux/ssh-from-windows.md

```diff
@@ -80,6 +80,12 @@ With the public key deployed on your Azure VM, and the private key on your local
 ssh -i ~/.ssh/id_rsa azureuser@10.111.12.123
 ```
 
+If you have never connected to this VM before you will be asked to verify the hosts fingerprint. It is tempting to simply accept the fingerprint presented, however, this exposes you to a possible person in the middle attack. You should always validate the hosts fingerprint. You only need to do this on the first time you connect from a client. To obtain the host fingerprint via the portal use the Run Command feature to execute the command `ssh-keygen -lf /etc/ssh/ssh_host_ecdsa_key.pub | awk '{print $2}'`.
+
+![Screenshot 2021-08-05 110213](https://user-images.githubusercontent.com/250240/128399040-f62cc497-c543-473b-8d37-79cce8c706d7.png)

```

Comment:
Can you put the screenshot here: https://github.com/MicrosoftDocs/azure-docs-pr/tree/master/articles/virtual-machines/linux/media/ssh-from-windows and then reference it there in the article? If you need help, let me know and I can step in!

Reply:
Done